### DISCUSSION TITLE
Rule to enforce Insert,modify,delete and rename triggers

### DISCUSSION THREAD

**Discussion by StefanMaron** [StefanMaron/BusinessCentral.LinterCop#13] [New Rule]
Similar to #11 record data modification triggers should always be called. If there is a reason to not do this, it should be documented

**Comment by Arthurvdv** [comment]
We now have the [LC0040](https://github.com/StefanMaron/BusinessCentral.LinterCop/wiki/LC0040) `Explicitly set the RunTrigger parameter on build-in methods.` rule which help to prevent the implicit usage of the false setting of the RunTigger on these actions.

Should we close this one?
